scrapers/teiid.py

```diff
@@ -31,14 +31,27 @@ def __init__(self, host, port, username, password):
         self._connections = {}
         self._last_query_dt = None
 
-    def get_connection(self, db_name, force_new=False, retry=None):
+    def get_connection(
+        self,
+        db_name,
+        force_new=False,
+        retry=None,
+        sslmode='verify-full',
+        sslrootcert='/etc/pki/tls/certs/ca-bundle.crt',
+    ):
         """
         Return an existing psycopg2 connection and establish it if needed.
 
         :param str db_name: the database name to get a connection to
         :kwarg bool force_new: forces a new database connection even if one already exists
         :kwarg int retry: the number of times to retry a failed connection. If this is not set,
             then the Teiid connection attempt will be repeated until it is successful.
+        :kwarg str sslmode: the Postgres SSL mode to use when connecting to Teiid. This defaults
```

Comment:
Maybe it's worth mentioning what other options are acceptable for this parameter?